tests/summarycode/data/classify/with_package_data.expected.json

```diff
@@ -500,15 +542,64 @@
       "is_media": false,
       "is_source": false,
       "is_script": false,
-      "package_data": [],
+      "package_data": [
+        {
+          "type": "maven",
+          "namespace": null,
+          "name": null,
+          "version": null,
+          "qualifiers": {},
+          "subpath": null,
+          "primary_language": "Java",
+          "description": null,
+          "release_date": null,
+          "parties": [],
+          "keywords": [],
+          "homepage_url": null,
+          "download_url": null,
+          "size": null,
+          "sha1": null,
+          "md5": null,
+          "sha256": null,
+          "sha512": null,
+          "bug_tracking_url": null,
+          "code_view_url": null,
+          "vcs_url": null,
+          "copyright": null,
+          "holder": null,
+          "declared_license_expression": null,
+          "declared_license_expression_spdx": null,
+          "license_detections": [],
+          "other_license_expression": null,
+          "other_license_expression_spdx": null,
+          "other_license_detections": [],
+          "extracted_license_statement": null,
+          "notice_text": null,
+          "source_packages": [],
+          "file_references": [],
+          "extra_data": {
+            "pom_properties": {
```

Comment:
Aren't these the namespace/name/version? Why are we having these in the extra-data instead of having these in the proper attributes (and then using these in the assembly)?

Reply:
I'm adding these in the main package attributes instead, so we have the correct PURL for these